servers/partner-ci/10.0/zabbix.yaml

```diff
@@ -0,0 +1,366 @@
+          skip_vote_failed:      '{skip_vote_failed}'
+          skip_vote_successfull: '{skip_vote_successfull}'
+          testrail_plan:         '{testrail_plan}'
+          testrail_plan_desc:    '{testrail_plan_desc}'
+          testrail_suite:        '{testrail_suite}'
+          testrail_suite_desc:   '{testrail_suite_desc}'
+          update_master:         '{update_master}'
+      - plugin-specific-parameters
+
+- defaults:
+    name: global
+    disabled: false
+    node: zabbix
+    <<: *test-parameters
+    scm:
+      - common-scm:
+          scm_submodule_disabled: '{scm_submodule_disabled}'
+          scm_wipe_workspace: '{scm_wipe_workspace}'
+          gerrit_repo: '{gerrit_repo}'
+          skip_vote_successfull: '{skip_vote_successfull}'
+          skip_vote_failed: '{skip_vote_failed}'
+    wrappers:
+      - test-wrapper
+    properties:
+      - test-property
+    builders:
+      - 10-get-refs
+      - get-props
+      - get-ubuntu-mirrors
+      - conditional-step:
+           condition-kind: shell
+           condition-command: |
+            [ "${{BUILD_PKG}}" == "true"       ]
+           steps:
+             - trigger-builds:
+                - project: '{fuel_release}.{zbx_extreme_net_repo}.build'
+                  current-parameters: false
+                  block: true
+                - project: '{fuel_release}.{zbx_emc_repo}.build'
+                  current-parameters: false
+                  block: true
+                - project: '{fuel_release}.{zbx_snmp_repo}.build'
+                  current-parameters: false
+                  block: true
+                - project: '{fuel_release}.{zbx_ext_repo}.build'
+                  current-parameters: false
+                  block: true
+      - copyartifact:
+          - project: '{fuel_release}.{zbx_extreme_net_repo}.build'
+            which-build: last-successful
+          - project: '{fuel_release}.{zbx_emc_repo}.build'
+            which-build: last-successful
+          - project: '{fuel_release}.{zbx_snmp_repo}.build'
+            which-build: last-successful
+          - project: '{fuel_release}.{zbx_ext_repo}.build'
+            which-build: last-successful
+      - shell:
+          !include-raw-escape: ../builders/common.prepare.sh
+      - shell:
+          !include-raw-escape: ../builders/zabbix.test.sh
+    publishers:
+      - runner-publisher:
+          properties_name: properties
+          email_to: '{email_to}'
+      - test-publisher:
+          reporter_name: '{reporter-name}'
+
+- job-template:
+    name: '{fuel_release}.{plugin_name}.{testgroup}'
+    description: 'The test "{testgroup}" for {fuel_release}.{plugin_name}'
+    concurrent: true
+
+- job-template:
+    name: '{fuel_release}.{plugin_name}.smoke.test'
+    description: 'The Smoke test for {fuel_release}.{plugin_name}'
+    concurrent: true
+
+- job-template:
+    name: '{custom-name}.qa'
+    description: 'The custom test for {fuel_release}.{plugin_name}'
```

Comment:
seems there was a problems
because u already commented job "custom qa" in above case